instant/flags.vim

```diff
@@ -66,6 +66,17 @@ call s:plugin.flags.clang_format_executable.AddCallback(
 " See http://clang.llvm.org/docs/ClangFormatStyleOptions.html for details.
 call s:plugin.Flag('clang_format_style', 'file')
 
+""
+" The path to the cmake-format executable. String, list, or callable that
+" takes no args and returns a string or a list.
+call s:plugin.Flag('cmake_format_executable', 'cmake-format')
+
+""
+" The path to the cmake-format configuration file for cmake-format to use.
+" See https://cmake-format.readthedocs.io/en/latest/installation.html for
```

Comment:
Looks like actually https://cmake-format.readthedocs.io/en/latest/configuration.html has the details and installation.html doesn't say much about it.